app/boards/arm/bluemicro52840_v1/bluemicro52840_v1.dts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2020 Pete Johanson, Derek Schmell
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+#include "arduino_pro_micro_pins.dtsi"
+
+/ {
+	model = "BlueMicro52840_V1";
+	compatible = "bluemicro52840,v1";
+
+	chosen {
+		zephyr,code-partition = &code_partition;
+		// zephyr,console = &uart0;
+		//zephyr,bt-mon-uart = &uart0;
+		//zephyr,bt-c2h-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		blue_led: led_0 {
+			gpios = <&gpio0 42 GPIO_ACTIVE_LOW>;
+			label = "Blue LED";
+		};
+	};
+
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&uart0 {
+	compatible = "nordic,nrf-uart";
+	current-speed = <115200>;
+	tx-pin = <6>;
+	rx-pin = <8>;
+};
+
+&i2c0 {
+	compatible = "nordic,nrf-twi";
+	sda-pin = <15>;
+	scl-pin = <17>;
+};
+
+&uart0 {
```

Comment:
You're overriding `uart0` *twice* in here, this one, and the one above. Which one is the right one? (if you inherited this from the nano board, we likely need to fix that too)

Reply:
Yeah it's directly from the nice_nano.

Reply:
Ok, then removed the first `uart0` node that is above this one please.

Reply:
Removed.